### DISCUSSION TITLE
New lvl50 Pets that stay out?

### DISCUSSION THREAD

**Discussion by TehNo0b** [Crowns-of-Power/CoP#17] [Ideas]
Can we get some new pets?  I've already made one new one for each color, they all stay out.  Some might want to say that only green should have pets that stay out, and then I remind you that theirs white.  Each pet casts spells related to their color, green still melees.  Having a non lvl50 pet when your lvl50 is like casting a supreme spell it doesn't even make sense.  

If people don't have the time to do this I can send a copy of the pet(s) and show you how to balance/adjust them.  As far as how/when to release them when in doubt add it to a token vendor, or make a questline.  No reason any/all new content has to always depend on a full new zone being released.. lots of things can just be a npc w/ quests.  Lets gooo

**Comment by Omega4200** [comment]
dont see what it would hurt id like to try it anyways.... pretty sure your right only white and green have lvl 50 pets i would like for the other colors to have lvl 50 pts im not sure abou tthem staying out all the time.... something worth checking into though im not sure it will hurt much green pets still eat through all the other pets fast really fast and dont take really any dmg

**Comment by gothrowhell2** [comment]
I was thinking about making new pets for red, blue, and maybe a new white instant pet. even though whites dont use their instant pets lol.

**Comment by TehNo0b** [comment]
Pets could be token vendor related as well.  We should have alot of pets, different pets to suit different needs and/or looks/skins.  Pets can come from quests, drops, or tokens.   I vote for all 3.. collect them all!
